### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by dargonjz**

### CODE REVIEW

**Comment by nkrone-csueb** [comment]
What exactly is this?

**Comment by dargonjz** [comment]
The file on github?
Its what I have so far on the gui
I'll tried to get more done

**Comment by nkrone-csueb** [comment]
Okay

File: zCRM.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+import javax.swing.event.ListSelectionListener;
+import java.io.File;
+import java.io.IOException;
+import javax.swing.*;
+
+import java.awt.event.*;
+
+public class zCRM {
+		
+//	private JFrame frame;
```
Comment (nkrone-csueb) [inline zCRM.java]: Remove all these

File: zCRM.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+		frame.setSize(400, 400);
+
+		JPanel p = new JPanel(new BorderLayout()); 	
+		frame.add(p);
+		    
+		Box infoBox = Box.createVerticalBox();
+		    
+		    
+		Box buttonBox = Box.createVerticalBox();
+		JButton addButton = new JButton("Add New DVD");
```
Comment (nkrone-csueb) [inline zCRM.java]: Remove references to DVD

File: zCRM.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
+
```
Comment (nkrone-csueb) [inline zCRM.java:1]: GUI should be separated out and placed inside the package (edu.cs401.group3.crm) 

GUI caller script (with the static main) should import and call the GUI library

**Review by nkrone-csueb — CHANGES_REQUESTED** [review]
You should have a issue created so we can keep track of what work is being done. Also, your commit message is to vague: "add files to upload" tells everyone nothing about this file aside from the fact that you're just uploading it to git.

A better commit message would be: "initial client gui" that way, when anyone first reviews the code, they have an understanding of what this file is attempting to accomplish.

Please review the suggestions I listed. Overall, it's a step in the right direction so good work.

File: zCRM.java (hunk 1 above)
Comment (nkrone-csueb) [reply zCRM.java]: review the commented out items that are not pertinent to the code. Commented out lines shouldn't be in code that's going to be merged.

File: zCRM.java (hunk 2 above)
Comment (nkrone-csueb) [reply zCRM.java]: This isn't a big deal really but if you're gonna copy paste code from previous work, at least remove references to it.

File: zCRM.java (hunk 3 above)
Comment (nkrone-csueb) [reply zCRM.java:1]: A better explanation would be having a Class called "CRMCLient" that calls the GUI library and runs it.

**Pull request closed without merging**